src/components/AdditionalInformation.jsx

```diff
@@ -162,11 +162,16 @@ const AdditionalInformation = props => {
 						</p>
 					</div>}
 				<div className = "cs-form-control" >
-					<input type="button" className="seButton" onClick={callPreviousForm} value="Previous" />
+					{<ButtonDisplay
```

Comment:
don't think you need the brackets around these.

Reply:
brackets have been murdered